### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by jreason** [spatie/laravel-options#11] [Ideas]
Would it be possible to add the following features?

->selected('ADMIN')->toHtml();

[
'label'=>'Admin', 'value'=>'ADMIN',
'label'=>'Basic', 'value'=>'BASIC'
]

outputs:
<option value="ADMIN" selected>Admin</option>
<option value="BASIC">Basic</option>

**Comment by rubenvanassche** [comment]
I don't think we want to have such a feature in this package. You can always convert the options into an array and then do the transformations.
